woocommerce/woocommerce
# Analyzer CLI: Refactor main file

### All Submissions:






### Changes proposed in this Pull Request:

`tools/code-analyzer/src/commands/analyzer/index.ts` was quickly becoming a very large class and the file was too difficult to manage, especially since database logic will be added. This PR breaks apart some methods into their own files for easier development.

### How to test the changes in this Pull Request:

1. See that Github Actions pass and no error occur in `Highlight templates and hooks changes`, which calls the CLI.

NOTE: the changes in `plugins/woocommerce/composer.json` are simply whitespace, so I don't think a changelog entry is required for `woocommerce` but the check fails anyways.

### Other information:




### FOR PR REVIEWER ONLY:
